.travis.yml

```diff
@@ -2,17 +2,12 @@ language: rust
 osx_image: xcode9.2
 
 rust:
-  - stable
-  - nightly
```

Comment:
Thanks for this! I think we'd like to keep nightly, and just replace `stable` with 1.26.